composer.json

```diff
@@ -34,6 +35,13 @@
     "require-dev": {
         "php-http/mock-client": "^1.5",
         "guzzlehttp/psr7": "^1.0",
-        "http-interop/http-factory-guzzle": "^1.0"
+        "http-interop/http-factory-guzzle": "^1.0",
+        "php-vcr/php-vcr": "^1.5",
+        "php-http/curl-client": "^2.2"
+    },
+    "config": {
+        "allow-plugins": {
+            "php-http/discovery": false
+        }
```

Comment:
This fixes a prompt that kept showing up when doing `make composer install`:

<details>
<summary>Prompt to remove</summary>

<img width="1003" alt="Screenshot 2023-04-12 at 4 25 13 PM" src="https://user-images.githubusercontent.com/8046636/231419528-74c93d04-5228-4953-9a33-d0557662b848.png">

</details>

Setting it to `false` so that it won't install any other plugins suggested and we can stick to the composer dependencies